integration-libs/cdc/manage-users/manage-users-button/cdc-list.component.html

```diff
@@ -0,0 +1,96 @@
+<!--copied from core module and altered create button attributes-->
```

Comment:
change the enclosing folder name from `manage-users-button` to `manage-users-list`

Reply:
done